components/Inspector.tsx

```diff
@@ -69,7 +69,7 @@ export default function Inspector(props: HTMLProps<HTMLDivElement>) {
           onClick={() => setPane(PANE.ABOUT)}
           badge={newCommitsCount > 0}
         >
-          About
+          News
```

Comment:
I renamed the tab to just "News", but I'd rather drop this list altogether.

We can replace it with:

_Last updated [3 days ago](https://github.com/npmgraph/npmgraph/commits/main/)_

or just a plain link when the changes are not recent:

_[Updates](https://github.com/npmgraph/npmgraph/commits/main/)_

Reply:
With the sponsor button and github link moving to the start page, I'm fine with killing this tab.  Showing the github activity was just an experiment to see if it had a noticeable impact on engagement / sponsorships.  (Spoiler: It didn't.)

Reply:
Re: "Last updated" or "Updates", I'm fine with just removing that part of things altogether.  That information is available on Github for anyone that really cares.

Reply:
<img width="425" alt="Screenshot 7" src="https://github.com/user-attachments/assets/bc4b137b-2541-48ed-b2cf-214d544a7392">

Reply:
<img width="425" alt="Screenshot" src="https://github.com/user-attachments/assets/23863e80-08dc-4c0f-b48c-bbfc2372776f">